docs/source/scancode-toolkit/cli-reference/core-options.rst

```diff
@@ -0,0 +1,282 @@
+modified by using this ``--license-score`` option.
+
+Some important INTEGER values of the ``--license-score INTEGER`` option:
+
+- **0**     - Default and Lowest Value, All matches are reported.
+- **100**    - Highest Value, Only licences with a much better match are reported
+
+Here, a bigger number means a better match, i.e. setting a higher license score translates to a
+higher threshold for matching licenses (with equal or less number of license matches).
+
+An example usage::
+
+    ./scancode -clieu --json-pp sample.json samples --license-score 70
+
+Here's the license results on setting the integer value to 100, vs the default value 0. This is
+visualized using ScanCode workbench in the License Info Dashboard.
+
+.. list-table:: License scan results of Samples Directory.
+
+    * - .. figure:: data/core_lic_score_0.png
+
+           License Score 0 (Default).
+
+      - .. figure:: data/core_lic_score_100.png
+
+           License Score 100.
+
+``--license-text`` Options
+--------------------------
+
+The ``--license-text`` option includes in the scan result, the matched text for the detected
+license, under the attribute "matched text".
+
+An example matched text included in the results is as follows::
+
+    "matched_text":
+     "  This software is provided 'as-is', without any express or implied
+     warranty.  In no event will the authors be held liable for any damages
+     arising from the use of this software.
+     Permission is granted to anyone to use this software for any purpose,
+     including commercial applications, and to alter it and redistribute it
+     freely, subject to the following restrictions:
+     1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+     2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+     3. This notice may not be removed or altered from any source distribution.
+
+     Jean-loup Gailly        Mark Adler
+     jloup@gzip.org          madler@alumni.caltech.edu"
+
+The file in which this license was detected: ``samples/arch/zlib.tar.gz-extract/zlib-1.2.8/zlib.h``
+License name: "ZLIB License"
+
+``--license-url-template`` Options
+----------------------------------
+
+The ``--license-url-template`` option sets the template URL used for the license reference URLs.
+
+The default template URL is : [https://enterprise.dejacode.com/urn/urn:dje:license:{}]
+In a template URL, curly braces ({}) are replaced by the license key.
+
+So, by default the license referance URL points to the dejacode page for that license.
+
+A scan example using the ``--license-url-template TEXT`` option ::
+
+    ./scancode -clpieu --json-pp sample_lic_url_template.json samples --license-url-template https://opensource.org/licenses/{}
```

Comment:
That's a great example but the likeliness that a ScanCode TK license key is the same as a the opensource.org key is super low IMHO.
Could you instead use an example using this base URL `https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses` ? and that would point to the .LICENSE file?

Reply:
You are right. I'll update this.

Reply:
This one is updated.